packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3644,15 +3660,25 @@ class InputDecorationTheme with Diagnosticable {
        assert(filled != null),
        assert(alignLabelWithHint != null);
 
+  /// The style to use for [InputDecoration.labelText] when the label is on top
+  /// of the input field.
+  ///
+  /// When the [InputDecoration.labelText] is floating above the input field,
+  /// the text uses the [floatingLabelStyle] instead.
+  ///
+  /// If null, defaults to a value derived from the base [TextStyle] for the
+  /// input field and the current [Theme].
+  final TextStyle? labelStyle;
+
   /// The style to use for [InputDecoration.labelText] when the label is
   /// above (i.e., vertically adjacent to) the input field.
   ///
   /// When the [InputDecoration.labelText] is on top of the input field, the
-  /// text uses the [hintStyle] instead.
+  /// text uses the [labelStyle] instead.
   ///
   /// If null, defaults to a value derived from the base [TextStyle] for the
   /// input field and the current [Theme].
-  final TextStyle? labelStyle;
+  final TextStyle? floatingLabelStyle;
```

Comment:
Same comment as earlier about the default value being the value of [labelStyle].